Core/Net/Net.cs

```diff
@@ -99,6 +100,7 @@ public static string Download(string url, out string etag, string filename = nul
             catch (Exception ex)
             {
                 log.InfoFormat("Download failed, trying with curlsharp...");
+                Console.WriteLine("Download failed, trying with curlsharp...");
```

Comment:
Please don't push these. We don't want debugging output in `Core.Net`.

Reply:
They are definitely not intended to be merged. However, I want them to be run on the build server, as I am struggling to reproduce the failed unit test locally.

Reply:
Ahh, OK. Carry on then.